gulpfile.babel.js

```diff
@@ -11,6 +11,12 @@ const $ = require('gulp-load-plugins')();
 const flexibility = require('postcss-flexibility')();
 const spawnSync = require('child_process').spawnSync;
 
+/*
+Note on deprecated deps (25.06.18):
+During `npm install` step you can see some warnings,
+related to deprecated deps. See issue #353 for more details.
```

Comment:
`See issue for more details` is too much for this simple idea.

Let's reflect this idea with few words without links:
```
You've seen warning because of deprecated deps during npm install.
It's recursive deps of actual packages, that we are using.
So, it's nothing we can do.
```

And i've got idea. To do this comment more useful, let's print this text at the beginning of this file. You should use `console.log`, as i know